src/Internal/Deserialization/FromBytes.purs

```diff
@@ -133,6 +135,12 @@ instance FromBytes Value where
 instance FromBytes VRFKeyHash where
   fromBytes' = fromBytesImpl "VRFKeyHash"
 
+instance FromBytes Vkeywitness where
+  fromBytes' = fromBytesImpl "Vkeywitness"
+
+instance FromBytes Vkeywitnesses where
```

Comment:
This type does not have from_bytes method

Reply:
Both of these do have `from_bytes`. I have tested `Vkeywitnesses`, so that one has one for sure, but the other one does too -- see the [typings](https://www.npmjs.com/package/@emurgo/cardano-serialization-lib-nodejs?activeTab=code) at line 8495. Or [the source](https://github.com/Emurgo/cardano-serialization-lib/blob/cac56c5b1493aeec2c15d71533f5f6252221f604/rust/src/crypto.rs#L551-L584).

Reply:
True, I was looking at an older version that we use.